query-parser/src/parse.rs

```diff
@@ -391,6 +414,51 @@ def_parser!(Find, query, FindQuery, {
         })
 });
 
+pub struct Bind;
+
+def_matches_plain_symbol!(Bind, placeholder, "_");
+
+def_matches_plain_symbol!(Bind, ellipsis, "...");
```

Comment:
Just use `Find.ellipsis()`. `Find` is the catch-all top-level parser struct here.